libnmstate/nm/active_connection.py

```diff
@@ -127,16 +127,22 @@ def is_active(self):
         if self.state == nm_acs.ACTIVATED:
             return True
         elif self.state == nm_acs.ACTIVATING:
+            ac_state_flags = nm_ac.get_state_flags()
```

Comment:
I think this should be `nm_acs.get_state_flags()`, just a typo.

Reply:
Thanks.